text/000-cli-redesign.md

```diff
@@ -1 +1,114 @@
+
+We’re going to create a CLI style guide and then implement that style guide, which includes redesigning error messages, introducing `gatsby --quiet` as a default mode and make more extensive use of `gatsby --verbose` flag. To make the CLI easier to design and improve, we’ll implement Ink. To optimize the CLI design, we’ll use the data being collected through our telemetry setup, which collects data about CLI usage (opt out is possible).
+
+# Basic example
+
+If the proposal involves a new or changed API, include a basic code example.
+Omit this section if it's not applicable.
+
+Note: [Ink](https://www.npmjs.com/package/ink), an npm package that give us the use of React for CLIs, will make it easier to design the CLI with consistency. Just as React helps  web UIs have consistent design, Ink will provide the same thing for our CLI.
+
+# Motivation
+## Why are we doing this?
+We have found many areas where the Gatsby CLI is doing well and many other areas where it is lacking. Here are resources that informed our knowledge of problems that Gatsby CLI has and possible ways to solve those problems:
+- [Evaluation of other CLI tools](https://github.com/gatsbyjs/gatsby/issues/12951)
+- [Nielsen Norman Group 10 usability heuristics](https://www.nngroup.com/articles/ten-usability-heuristics/)
+- [Nielsen Norman Group error message design](https://www.nngroup.com/articles/error-message-guidelines/)
+- [Heroku CLI style guide](https://devcenter.heroku.com/articles/cli-style-guide)
+- [12 Factor CLI Apps](https://medium.com/@jdxcode/12-factor-cli-apps-dd3c227a0e46)
+- [User journey through the Gatsby CLI](https://whimsical.co/2PxMcRGE63bwk1Ayc3emAB) (informed by usability interviews and the Gatsby core team’s observations of what issues are most common regarding the CLI)
+
+## What use cases does it support? 
+Developers who use the Gatsby CLI to start, develop, and build Gatsby sites.
+
+## What is the expected outcome?
+The expected outcome is that the Gatsby CLI will adhere to common sense usability and accessibility standards and will also help people start, develop, and build Gatsby sites faster and with less frustration.
+
+Here are the “user can” statements we want to be true after this redesign:
+Contributors and Ink team members can decide which changes to merge or not through usability heuristics and style principles
+Users can know what an error is and how to solve it
+Users can know the current state of the system
+Users can know when they need to take action and how to take that action
+
+## Data collection outcomes
+See this [RFC on Error surveying & improvement](https://github.com/gatsbyjs/rfcs/pull/37)
+
+## Objectives
+- Increase the rate of adoption - more Gatsby CLI users!
+- Make sure users can reach their goals through the CLI
+- CLI should embody the principles in the CLI style guide
+
+## Anti-goals
+- losing Gatsby users
```

Comment:
Should we capitalize these points like in `Objectives` section ?

Reply:
Sure